src/styles/blocks/header.scss

```diff
@@ -26,10 +26,15 @@
     color: #060b35;
     list-style: none;
     line-height: 60px;
+    transition: $transition-time;
```

Comment:
it's better to indicate which property you animated.

Reply:
agree, add style into transition

Reply:
ops